src/ledger/LedgerHashUtils.h

```diff
@@ -52,10 +52,9 @@ getAssetHash(T const& asset)
     }
     case stellar::ASSET_TYPE_POOL_SHARE:
     {
-        size_t res = asset.type();
```

Comment:
This is a functional change:  you're removing the asset type from contributing to the hash. Not sure you want to do that.

Reply:
Oh, never mind, I expanded the diff a bit further and saw it's set above as well.